StreamChatUITestsAppUITests/MockServer/MessagingResponses.swift

```diff
@@ -0,0 +1,95 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChat
+import Swifter
+
+extension StreamMockServer {
+    
+    func configureMessagingEndpoints() {
```

Comment:
Same here with `MessagingResponses.swift`, this file is out of date. No trace in the source code. `MessageResponses.swift` is the right one.

Reply:
Yeah, I can't see this either in my Xcode, it's the correct version

Reply:
Seems like, for some reason, your PR brings these files back, but without reference in Xcode. Could you please manually remove `StreamChatUITestsAppUITests/MockServer/MessagingResponses.swift` and `StreamChatUITestsAppUITests/MockServer/MessageDetails.swift` in Finder?